ibc/host/keys.go

```diff
@@ -0,0 +1,285 @@
+package host
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Store key prefixes for IBC
+const (
+	KeyClientStorePrefix       = "clients"
+	KeyClientState             = "clientState"
+	KeyConsensusStatePrefix    = "consensusStates"
+	KeyConnectionPrefix        = "connections"
+	KeyChannelEndPrefix        = "channelEnds"
+	KeyChannelPrefix           = "channels"
+	KeyPortPrefix              = "ports"
+	KeySequencePrefix          = "sequences"
+	KeyChannelCapabilityPrefix = "capabilities"
+	KeyNextSeqSendPrefix       = "nextSequenceSend"
+	KeyNextSeqRecvPrefix       = "nextSequenceRecv"
+	KeyNextSeqAckPrefix        = "nextSequenceAck"
+	KeyPacketCommitmentPrefix  = "commitments"
+	KeyPacketAckPrefix         = "acks"
+	KeyPacketReceiptPrefix     = "receipts"
+)
+
+// DISCUSSION: Do we need both paths and keys with the ApplyPrefix and RemovePrefix functions?
```

Comment:
To be clear the difference between the Path and Keys is actually that the Path is without the prefix and the Key is with the prefix + as a []byte.

The path is intended to be used with the `ApplyPrefix()` function whereas the Key can be used as a hardcoded path. IMO it would probably be better to remove the Path functions and keep the keys as every path will be known before hand and are not chosen.

Reply:
Totally up to you. I feel like we'll only know what's more/less useful as we dive deeper into the implementation